Update docs for new UI + some rewording (#105)
yaptide/docs: docs/geometry.md

Both are centered around (0,0,0) point.
You can use drag operation by clicking and dragging on the 3-D projection view. This would help to adjust the view, so the collimator is visible.

![Cylinders](assets/geometry/cylinders.png)

## Adding new zone

With the figures defined above, we can construct the zones which will be used in the simulation.
Here we follow the naming of SHIELD-HIT12A, as Fluka calls the zones "regions".

Lets start with adding a zone which will represent the large box filled with air.
This can be done in a similar way as adding the figures, either from upper left menu in the Editor tab
or by going to Geometry tab in right menu and expanding ZONES section in GEOMETRY tab.

![Add Boolean Zone from Object Menu](assets/geometry/menu_boolean_zone_add.png)
![Add Boolean Zone from ZONES section](assets/geometry/boolean_zone_add.png)

> [!NOTE]
> A special "World Zone" which represents the whole simulation environment is defined by default, and can't be removed.

### Air zones

Zones describe a volume of space by combining multiple figures using boolean operations to define shape,
and define physical material that the volume is made of.

> [!IMPORTANT]

> > Defining zone operations requires special considerations. Each volume of space should belong to exactly

Reviewer: ```suggestion
> Defining zone operations requires special considerations. Each point in space should belong to exactly
```
